tests/test-sources/plugins/lsp/lsp-status.nix

```
@@ -0,0 +1,26 @@
+{
+  empty = {
```

```suggestion
  empty = {
    plugins.lsp.enable = true;
    
```

(Same for `defaults` test)